js/sticky-bar.js

```diff
@@ -13,14 +14,16 @@ function StickyBar(selector) {
 }
 
 StickyBar.prototype.toggle = function() {
-  var scrollTop = this.$body.scrollTop();
-  if (scrollTop >= this.offset + this.triggerOffset) {
-    var height = this.$bar.outerHeight();
-    this.$bar.addClass('is-stuck');
-    this.$body.css('padding-top', height);
-  } else if (scrollTop < this.offset) {
-    this.$bar.removeClass('is-stuck');
-    this.$body.css('padding-top', this.defaultBodyPadding);
+  if (helpers.isLargeScreen()) {
```

Comment:
nit picky, but I prefer a "fail fast" style rather than the entire body of the function in an `if`... e.g.
```javascript
if (!helpers.isLargeScreen()) {
  return;
}

// function code
```